src/tooling_webserver.nim

```diff
@@ -40,12 +41,12 @@ router routes:
     let response = %*
         {
             "exit_status": exit_status,
-            "result": nil
+            "output_files": %*{}
```

Comment:
I've renamed this to `output_files` which IMHO better indicate what its contents are. It also fits with the input/output dir terminology used to describe the tooling interfaces.

Reply:
Cool. I've commented on this [here](https://github.com/exercism/tooling-orchestrator/pull/14#pullrequestreview-492214423) but I'm happy with it 🙂